workshop/01-link-datasource.md

```diff
@@ -0,0 +1,56 @@
+# Link your persistent datasource
+
+We're going to be setting up model objects to be transferred across our API, but to keep track of them, we'll need a persistent datasource to stash them in.
+
+Go to your command line in the root directory we've been working in, type `lb datasource`, and you'll see the following:
```

Comment:
This is an example. I have been avoiding the word you unless nothing else makes sense. It is super minor, but has just been a consistent style I've taken. So in this line, I would say ", and we'll see the following:"

Again, this is trivial. I think when Erin pointed it out for me, it was a more obvious voice change, so do what you feel right.